File: official/docs/golang/v3/addresses/create-and-verify.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v3"
)

func main() {
	client := easypost.New("EASYPOST_API_KEY")

	address, _ := client.CreateAndVerifyAddress(
		&easypost.Address{
			Street1: "417 Montgomery Street",
			Street2: "FL 5",
			City:    "San Francisco",
			State:   "CA",
			Zip:     "94104",
			Country: "US",
			Company: "EasyPost",
			Phone:   "415-123-4567",
		},
		&easypost.CreateAddressOptions{},
	)

	fmt.Println(address)
}

File: official/docs/csharp/v4/addresses/create-and-verify.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            Address address = await client.Address.CreateAndVerify(
                new Dictionary<string, object>
                {
                    { "street1", "417 Montgomery Street" },
                    { "street2", "FL 5" },
                    { "city", "San Francisco" },
                    { "state", "CA" },
                    { "zip", "94104" },
                    { "country", "US" },
                    { "company", "EasyPost" },
                    { "phone", "415-123-4567" },
                }
            );

            Console.WriteLine(JsonConvert.SerializeObject(address, Formatting.Indented));
        }
    }
}
